robottelo/products.py

```diff
@@ -763,7 +763,14 @@ def need_subscription(self):  # type: () -> bool
     def organization(self):
         return self._org
 
-    def add_item(self, item):  # type: (BaseRepository) -> None
+    def add_item(self, item) -> None:
+        """
+        Add repository to collection
+
+        :type item: BaseRepository
+        :param item: Item to add
+        :returns: None
+        """
```

Comment:
```suggestion
       def add_item(self, item):
        """
        Add repository to collection

        :param BaseRepository item: Item to add
        :returns: None
        """
```

Reply:
I would swear I have tried this version before and it was giving me some error, but now I don't see it anymore.
Done in aacdd4cbcb8988ab9944cfbd3972a8bfab80021e.